devicetypes/smartthings/smartweather-station-tile.src/smartweather-station-tile.groovy

```diff
@@ -271,7 +271,8 @@ def poll() {
         }
 
         // Alerts
-        def alerts = getTwcAlerts(zipCode)
+        def zipToLocation = getTwcLocation("$zipCode").location
```

Comment:
This is redundant. We already got the location on line 231 in the variable `loc`

Reply:
Good catch!